content/pt-br/docs/reference/glossary/persistent-volume.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Volume Persistente
+id: persistent-volume
+date: 2018-04-12
+full_link: /pt-br/docs/concepts/storage/persistent-volumes/
+short_description: >
+  Um objeto de API que representa uma parte do armazenamento no cluster. Disponível como um recurso conectável geral que persiste além do ciclo de vida de qualquer pod individual.
+
+aka: 
+tags:
+- core-object
+- storage
+---
+ Um objeto de API que representa uma parte do armazenamento no cluster. Disponível como um recurso conectável geral que persiste além do ciclo de vida de qualquer {{< glossary_tooltip text="Pod" term_id="pod" >}} individual.
+
+<!--more--> 
+
+PersistentVolumes (PVs) fornecem uma API que abstrai detalhes de como o armazenamento é fornecido de como ele é consumido.
```

Comment:
Sounds better to me "fornecido a partir de como ele é consumido" instead of "fornecido de como ele é consumido."